tests/android/browser.spec.ts

```diff
@@ -32,12 +32,20 @@ test('androidDevice.launchBrowser', async function({ androidDevice }) {
   await context.close();
 });
 
-test('androidDevice.launchBrowser should pass args with spaces', async ({ androidDevice }) => {
-  const context = await androidDevice.launchBrowser({ args: ['--user-agent=I am Foo'] });
-  const page = await context.newPage();
-  const userAgent = await page.evaluate(() => navigator.userAgent);
-  await context.close();
-  expect(userAgent).toBe('I am Foo');
+test('androidDevice.launchBrowser should treat args correctly', async ({ androidDevice }) => {
```

Comment:
- We need to escape the double quotes before we send it to the Android backend (linux bash)